source/transactions/tests/retryable-abort-errorLabels.yml

```diff
@@ -13,18 +13,18 @@ tests:
       configureFailPoint: failCommand
       mode: { times: 2 }
       data:
-        failCommands: ["abortTransaction"]
-        errorCode: 112 # WriteConflict, not a retryable error code
-        errorLabels: ["RetryableWriteError"] # Override server behavior: send RetryableWriteError label with non-retryable error code
+          failCommands: ["abortTransaction"]
+          errorCode: 112 # WriteConflict, not a retryable error code
+          errorLabels: ["RetryableWriteError"] # Override server behavior: send RetryableWriteError label with non-retryable error code
```

Comment:
Can you include the SPEC ticket that reminds us to remove this?

Reply:
That label is actually part of the test, and should still be there after the other ones get removed.